phoenix-core/src/test/java/org/apache/phoenix/util/TestUtil.java

```diff
@@ -934,6 +934,22 @@ public static void dumpIndexStatus(Connection conn, String indexName) throws IOE
     }
     }
 
+    public static void printResultSet(ResultSet rs) throws SQLException {
+        while(rs.next()){
+            StringBuilder builder = new StringBuilder();
+            int columnCount = rs.getMetaData().getColumnCount();
+            for(int i = 0; i < columnCount; i++) {
+                Object value = rs.getObject(i+1);
+                String output = value == null ? "null" : value.toString();
+                builder.append(output);
+                if(i + 1 < columnCount){
+                    builder.append(",");
+                }
+            }
+            System.out.println(builder.toString());
```

Comment:
Remove stdout prints?

Reply:
This was a test function (that i've written over 20 times now) that i intended to call print on.  I can consider calling a logger instead or returning a string.  Thoughts?

Reply:
ok disregard.